pkg/selector/comparators.go

```diff
@@ -140,6 +142,50 @@ func getTotalGpuMemory(gpusInfo *ec2.GpuInfo) *int64 {
 	return gpusInfo.TotalGpuMemoryInMiB
 }
 
+func getGPUManufacturers(gpusInfo *ec2.GpuInfo) []*string {
+	if gpusInfo == nil {
+		return nil
+	}
+	var manufacturers []*string
+	for _, info := range gpusInfo.Gpus {
+		manufacturers = append(manufacturers, info.Manufacturer)
+	}
+	return manufacturers
+}
+
+func getGPUModels(gpusInfo *ec2.GpuInfo) []*string {
+	if gpusInfo == nil {
+		return nil
+	}
+	var models []*string
+	for _, info := range gpusInfo.Gpus {
+		models = append(models, info.Name)
+	}
+	return models
+}
+
+func getInferenceAcceleratorManufacturers(acceleratorInfo *ec2.InferenceAcceleratorInfo) []*string {
+	if acceleratorInfo == nil {
+		return nil
+	}
+	var manufacturers []*string
+	for _, info := range acceleratorInfo.Accelerators {
+		manufacturers = append(manufacturers, info.Manufacturer)
+	}
+	return manufacturers
+}
+
+func getInferenceAcceleratorModels(acceleratorInfo *ec2.InferenceAcceleratorInfo) []*string {
+	if acceleratorInfo == nil {
+		return nil
+	}
+	var models []*string
+	for _, info := range acceleratorInfo.Accelerators {
+		models = append(models, info.Manufacturer)
```

Comment:
This looks to be adding `.Manufacturer` to the `models` array, but that's also what `getInferenceAcceleratorManufacturers()` is doing. Is that correct?

Reply:
oooooooooops

Reply:
fixed